packages/storybook/stories/components/toggle/toggle.stories.ts

```diff
@@ -0,0 +1,194 @@
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { defineCustomElement } from '@ovhcloud/ods-components/dist/components/ods-toggle';
+import { html } from 'lit-html';
+import { CONTROL_CATEGORY, orderControls } from '../../control';
+
+defineCustomElement();
+
+const meta: Meta = {
+  title: 'ODS Components/Form/Toggle',
+  component: 'ods-toggle',
+};
+
+export default meta;
+
+export const Demo: StoryObj = {
+  render: (arg) => html`
+<ods-toggle class="my-toggle-demo"
+  is-disabled=${arg.isDisabled}
+  is-required=${arg.isRequired}
+  with-label=${arg.withLabel}>
+</ods-toggle>
+<style>
+  .my-toggle-demo::part(slider) {
+    ${arg.CustomCssSlider}
+  }
+
+  .my-toggle-demo::part(slider)::before {
+    ${arg.CustomCssSlider}
+  }
+
+  .my-toggle-demo::part(label) {
+    ${arg.CustomCssLabel}
+  }
+
+  .my-toggle-demo::part(slider checked) {
+    ${arg.CustomCssSliderChecked}
+  }
+
+  .my-toggle-demo::part(slider checked)::before {
+    ${arg.CustomCssSliderChecked}
+  }
+
+  .my-toggle-demo::part(label checked) {
+    ${arg.CustomCssLabelChecked}
+  }
+</style>
+  `,
+  argTypes: orderControls({
+    CustomCssLabel: {
+      control: 'text',
+      description: 'Set a custom style properties for the slider. Example: "color: green;"',
+      table: {
+        category: CONTROL_CATEGORY.design,
+        defaultValue: { summary: 'ø' },
+        type: { summary: 'string' },
+      },
+    },
+    CustomCssLabelChecked: {
+      control: 'text',
+      description: 'Set a custom style properties for the slider. Example: "color: black;"',
+      table: {
+        category: CONTROL_CATEGORY.design,
+        defaultValue: { summary: 'ø' },
+        type: { summary: 'string' },
+      },
+    },
+    CustomCssSlider: {
+      control: 'text',
+      description: 'Set a custom style properties for the slider. Example: "background-color: red;"',
+      table: {
+        category: CONTROL_CATEGORY.design,
+        defaultValue: { summary: 'ø' },
+        type: { summary: 'string' },
+      },
+    },
+    CustomCssSliderChecked: {
+      control: 'text',
+      description: 'Set a custom style properties for the slider. Example: "background-color: blueviolet;"',
+      table: {
+        category: CONTROL_CATEGORY.design,
+        defaultValue: { summary: 'ø' },
+        type: { summary: 'string' },
+      },
+    },
+    CustomCssSliderCircle: {
+      control: 'text',
+      description: 'Set a custom style properties for the slider. Example: "background-color: blue;"',
+      table: {
+        category: CONTROL_CATEGORY.design,
+        defaultValue: { summary: 'ø' },
+        type: { summary: 'string' },
+      },
+    },
+    CustomCssSliderCircleChecked: {
+      control: 'text',
+      description: 'Set a custom style properties for the slider. Example: "background-color: black;"',
```

Comment:
update each description to be more precise about which element is actually concerned by this customization